src/app/grocery-total/grocery-total.component.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  <h3>Total</h3>
+</div>
+
+<div>
+  <span class="total-number">{{calcTotal()}}</span>
```

Comment:
You could have used the `currency` pipe to show the currency symbol and handle the correct decimal place